src/popup/pages/SignTransaction.vue

```diff
@@ -34,11 +37,30 @@ export default defineComponent({
       const { nodeNetworkId, getAeSdk } = useAeSdk();
       const { openDefaultModal } = useModals();
       const { setLoaderVisible } = useUi();
+      const { getLastActiveProtocolAccount } = useAccounts();
+
+      const aeSdk = await getAeSdk();
+      const activeAccountAddress = getLastActiveProtocolAccount(
+        PROTOCOLS.aeternity,
+      )?.address as Encoded.AccountAddress;
+
+      async function replaceCallerTx(tx: Encoded.Transaction) {
```

Comment:
I suggest to convert this function into pure function by adding 2nd argument which would take the new `callerId` and replace it.

Reply:
done